apps/sms/test/marionette/lib/messages.js

```diff
@@ -64,11 +64,20 @@ var ParticipantsAccessor = require('./views/participants/accessors');
 
         Participants: new ParticipantsAccessor(client),
 
-        launch() {
+        launch(doNotWaitForAppToBecomeReady) {
           client.switchToFrame();
           client.apps.launch(ORIGIN_URL);
           client.apps.switchToApp(ORIGIN_URL);
-          client.helper.waitForElement(SELECTORS.appReady);
+
+          // In 99.99% cases we'd like to wait for the app to become fully ready
```

Comment:
I'm still not sure why we should have 2 different way for waiting app launch. Is there any problem that we always wait for app ready?